src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1558,7 +1560,7 @@ GrpcLb::PendingPick* GrpcLb::PendingPickCreate(PickState* pick) {
   pp->grpclb_policy = this;
   pp->pick = pick;
   GRPC_CLOSURE_INIT(&pp->on_complete, &GrpcLb::OnPendingPickComplete, pp,
-                    grpc_schedule_on_exec_ctx);
+                    grpc_combiner_scheduler(combiner()));
```

Comment:
I think there is no harm to run the wrapper closure in the same combiner as the original one if we use `GRPC_CLOSURE_RUN` instead of `GRPC_CLOSURE_SCHED`, because we will need to grab the combiner at least once anyway.

Reply:
There certainly won't be any harm but there is a subtle performance implication. If this is a correctness issue, then I don't mind but if it's performance motivated then there is a slight discussion involved